opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -679,6 +683,11 @@ public static void main(String[] argv) {
                     "Run as quietly as possible. Sets logging level to WARNING.").execute(v ->
                     LoggerUtil.setBaseConsoleLogLevel(Level.WARNING));
 
+            parser.on("--repositorySearchThreads", "=number", Integer.class,
```

Comment:
This is mis-ordered and should appear after `--repository`

Reply:
fixed